### PR TITLE
[ONNX] Support embedding_renorm ONNX export

### PR DESCRIPTION
**Pull request by BowenBao**
Composite using ONNX operators for same logic from here https://github.com/pytorch/pytorch/blob/0a07488ed2c47765e337e290bd138c0e6e459cbd/aten/src/ATen/native/Embedding.cpp#L153

### CODE REVIEW

File: torch/onnx/symbolic_opset11.py [hunk 1]
```diff
@@ -894,6 +894,21 @@ def embedding_bag(g,
     return loop.node().output(), None, None, None
 
 
+@parse_args('v', 'v', 'f', 'f')
```
Comment (garymm) [inline torch/onnx/symbolic_opset11.py:897]: double quotes for consistency

File: torch/onnx/symbolic_opset11.py [hunk 2]
```diff
@@ -894,6 +894,21 @@ def embedding_bag(g,
...
+def embedding_renorm(g, weight, indices, max_norm, norm_type):
+    unique_indices = g.op("Unique", indices)
+    partial_weight = g.op("Gather", weight, unique_indices)
+    norm_type = "ReduceL2" if int(norm_type) == 2 else "ReduceL1"
+    partial_weight_norm = g.op(norm_type, partial_weight, axes_i=[1], keepdims_i=1)
+    partial_weight_norm_ = g.op("Add", partial_weight_norm, g.op("Constant", value_t=torch.tensor(1e-7)))
```
Comment (garymm) [inline torch/onnx/symbolic_opset11.py:903]: add a link to the source for 1e-7. In general constants like this should have some explanation.

File: torch/onnx/symbolic_opset11.py [hunk 3]
```diff
@@ -894,6 +894,21 @@ def embedding_bag(g,
     return loop.node().output(), None, None, None
 
 
+@parse_args('v', 'v', 'f', 'f')
+def embedding_renorm(g, weight, indices, max_norm, norm_type):
+    unique_indices = g.op("Unique", indices)
+    partial_weight = g.op("Gather", weight, unique_indices)
+    norm_type = "ReduceL2" if int(norm_type) == 2 else "ReduceL1"
```
Comment (garymm) [inline torch/onnx/symbolic_opset11.py:901]: PyTorch seems to accept 0 and 3 as valid norm types. I guess we only support 1 and 2?
If so let's make that explicit and raise an error if norm type is not 1 or 2.

**Review by garymm — APPROVED** [review]
Thanks for this!
Just a few minor changes, feel free to merge after addressing.
Can you please merge into master directly rather than onnx_ms_1?

File: torch/onnx/symbolic_opset11.py (hunk 2 above)
Comment (BowenBao) [reply torch/onnx/symbolic_opset11.py:903]: added comments

File: torch/onnx/symbolic_opset11.py (hunk 1 above)
Comment (BowenBao) [reply torch/onnx/symbolic_opset11.py]: done

File: torch/onnx/symbolic_opset11.py (hunk 3 above)
Comment (BowenBao) [reply torch/onnx/symbolic_opset11.py]: Good catch, added check to raise when not 1 or 2.

**Comment by BowenBao** [comment]
Thanks @garymm for review. Opened #72738 to replace this to avoid cross repo merge.

**Pull request closed without merging**